crates/telio-proxy/src/proxy.rs

```diff
@@ -262,6 +286,12 @@ impl StateEgress {
         Ok(map)
     }
 
+    fn get_replaced_sockets(&mut self) -> Result<EndpointMap, Error> {
```

Comment:
Nitpick: since it clears `self.replaced_sockets` maybe let's call it `take_replaced_sockets()`?

Reply:
Good suggestion! I was thinking about having a name with `take` in it but couldn't think of something, yet it was so simple!